src/PointwiseFunctions/Hydro/Tags.hpp

```diff
@@ -97,6 +97,14 @@ struct EquationOfState : EquationOfStateBase, db::SimpleTag {
   }
 };
 
+/// The inverse plasma beta \f$\beta^{-1} = b^2 / (2 p)\f$, where
+///// \f$b^2\f$ is the square of the comoving magnetic field amplitude
+///// and \f$p\f$ is the fluid pressure.
+template <typename DataType>
```

Comment:
Can you add a test for this to `tests/Unit/PointwiseFunctions/Hydro/Test_Tags.cpp`?